conda_forge_tick/auto_tick.xsh

```diff
@@ -475,7 +475,12 @@ def add_rebuild_migration_yaml(migrators, gx, package_names, yaml_contents,
     for node, node_attrs in gx.nodes.items():
         attrs = node_attrs['payload']
         meta_yaml = attrs.get("meta_yaml", {}) or {}
-        bh = get_requirements(meta_yaml, run=False, build=False, host=True) or get_requirements(meta_yaml, build=Truerun=False, host=False)
+        if ('strong' in meta_yaml.get('build', {}) or 
```

Comment:
This is looking at strong run_exports of the downstream package? You need to check in the upstream package

Reply:
I'm confused.

Reply:
Actually can we leave this off for now? It seems that strong exports are limited to the compilers and most likely aren't in the python graph

Reply:
Sure, but can you open an issue so that it is tracked?

Reply:
https://github.com/regro/cf-scripts/issues/677